rootbeerlib/build.gradle

```diff
@@ -2,49 +2,46 @@ apply plugin: 'com.android.library'
 
 android {
     compileSdkVersion compileSdkVer
-    buildToolsVersion buildToolsVer
-
+    buildToolsVersion '28.0.3'
```

Comment:
can we stick to using the buildToolsVer and increment that to 28.0.3?

Reply:
Changed buildToolsVersion to buildToolsVer.
Thank you for your confirmation.